Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.UnitTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,2 @@
+
+[assembly: Xunit.CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Because the unit tests involve setting environment variables for Amazon.Lambda.RuntimeSupport I disabled test parallelization to avoid flakiness running the tests.